src/TeamIcon.scss

```diff
@@ -0,0 +1,34 @@
+
+
+article {
+  border: 2px solid;
+  border-radius: 15px;
+  height: 250px;
+  width: 250px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  margin: 10px;
+  background-image: radial-gradient(circle, white, black);
+  opacity: 0.7;
+  
+}
+
+article:hover {
+  cursor: pointer;
+  opacity: 1;
+}
+
+img {
+  max-height: 175px;
+  max-width: 175px;
+  /*background: rgba(255,255,255,0.8);*/
```

Comment:
need to take this out, we have the issue created to look into solving this at a later time